plugins/outputs/influxdb_v3/README.md

```diff
@@ -0,0 +1,113 @@
+# InfluxDB v3.x Output Plugin
+
+This plugin writes metrics to a [InfluxDB v3.x][influxdb_v3] instance via HTTP.
```

Comment:
I cannot find any documentation about a specific v3 HTTP API:

https://docs.influxdata.com/influxdb3/cloud-serverless/reference/api/
https://docs.influxdata.com/influxdb3/cloud-serverless/write-data/

Could you point me where to look?

Reply:
Currently the only documentation is the code. Sorry! Take a look at the [Getting Started guide](https://docs.influxdata.com/influxdb3/core/get-started/). Will add a better link once the documentation is out.